packages/gatsby-source-airtable/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "gatsby-source-airtable",
+  "description": "Gatsby Airtable Source with Linked Nodes, Multiple Tables and File Nodes.",
+  "version": "2.3.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsby-uc/plugins.git",
+    "directory": "packages/gatsby-plugin-airtable"
+  },
+  "author": "Jacob Bolda <me@jacobbolda.com>",
+  "license": "MIT",
+  "main": "index.js",
+  "files": [
+    "/gatsby-node.js"
+  ],
+  "keywords": [
+    "airtable",
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source",
+    "gatsby-source-airtable"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\""
```

Comment:
This should execute a test if we want them to run on our current CI config.

Reply:
The "tests" we had were running it within the examples. There are no unit tests.

Reply:
I totally understand the separation. Probably how we should do the rest of the integration tests. I can work on figuring this out but we just need to change CI to run integration tests as a separate CI job. Part of me thinks this should be a separate PR, but I also want it for this so we actually get the tests passing for this PR. Thoughts?

Reply:
So Just looking through the integration tests and I'm not seeing any "test" scripts there either. How are you expecting to execute tests?

Reply:
I kind of feel like it is a slippery slope. If you start running the builds, you might run into another thing that should be fixed and a rabbit hole of things to update. I didn't include a change file so this should publish a new release. It feels like there is the opportunity there to do some updating separate from this PR.

Ah, so the examples don't have a test script either. They are pretty horribly styled examples that are structured to show one or many of the features of the source plugin. The idea is that the `gatsby build` would straight up fail if there was an issue. This was kind of the best path that I found to maintain it while working with v0, v1 and v2.

Reply:
Sounds good. I think I'm comfortable with this as is. Maybe we create an issue to resolve testing in the future.